internal/kibana/import_saved_objects/schema.go

```diff
@@ -0,0 +1,157 @@
+package import_saved_objects
+
+import (
+	"context"
+
+	"github.com/elastic/terraform-provider-elasticstack/internal/clients"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+// Ensure provider defined types fully satisfy framework interfaces
+var _ resource.Resource = &Resource{}
+var _ resource.ResourceWithConfigure = &Resource{}
+
+// var _ resource.ResourceWithConfigValidators = &Resource{}
+
+// func (r *Resource) ConfigValidators(context.Context) []resource.ConfigValidator {
+// 	return []resource.ConfigValidator{
+// 		resourcevalidator.Conflicting(
+// 			path.MatchRoot("create_new_copies"),
+// 			path.MatchRoot("overwrite"),
+// 			path.MatchRoot("compatibility_mode"),
+// 		),
+// 	}
+// }
+
```

Comment:
Can we remove it?

Reply:
Added a comment here. The kibana client doesn't support `create_new_copies` and `compatibility_mode` which we could add when https://github.com/elastic/terraform-provider-elasticstack/pull/372 is merged.